force-app/main/default/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -697,6 +703,11 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
             primaryConIdForAccount.put(acc.Primary_Contact__c, acc);
         }
 
+        // Clear cache to support real-time account model updates within the testing context.
+        if (Test.isRunningTest()) {
+            defaultRecTypeId = UTIL_CustomSettingsFacade.getSettings().Account_Processor__c;
+        }
+
```

Comment:
Why do we need to alter the logic in the test context?

Reply:
This change is relevant to [T-3904058](https://gus.lightning.force.com/lightning/r/ADM_Test_Scenario__c/a80B00000001ZduIAE/view). 

Given we are changing the default account model from household to administrative - the `defaultRecTypeId` was previously misreporting the value(because it is cached); This did not surface previously in our existing test suite because we haven't updated the default account model after setting it initially.

Reply:
I see, thanks for the explanation. Maybe instead of altering the logic here we could add `@TestVisible` annotation to `defaultRecTypeId` and change it from the test class? Just a suggestion
TDTM static variables is a problem, I saw similar problematic behavior in a few other places too

Reply:
Sure thing, that should also work ([reference commit](https://github.com/SalesforceFoundation/EDA/pull/1950/commits/905f6f23fa3b337318fa81e9ecf1bfd881e146a8))